Meta/Lagom/Tools/CodeGenerators/LibWeb/BindingsGenerator/IDLGenerators.cpp

```diff
@@ -130,6 +130,9 @@ CppType idl_type_name_to_cpp_type(Type const& type, Interface const& interface)
     if (type.name() == "BufferSource")
         return { .name = "JS::Handle<JS::Object>", .sequence_storage_type = SequenceStorageType::MarkedVector };
 
+    if (type.name() == "XMLHttpRequestBodyInit")
+        return { .name = "Fetch::XMLHttpRequestBodyInit", .sequence_storage_type = SequenceStorageType::MarkedVector };
```

Comment:
I thought I made a comment here, but it seems to be lost: why is this needed? :thinking:

Reply:
Our IDLGenerator needs it after changing the signature of `XHR::send()` method from `Optional<Fetch::XMLHttpRequestBodyInit>` to `Optional<Variant<JS::Handle<DOM::Document>, Fetch::XMLHttpRequestBodyInit>>`.

Without it we get the `Optional<Variant<JS::Handle<DOM::Document>, Fetch::XMLHttpRequestBodyInit>>` and hits a TODO.

Reply:
Kinda weird that it can figure out "nullable type" but needs help for "nullable type that's either this or that", but let's fix that some other day :)